README.md

```diff
@@ -3,7 +3,7 @@
 sccache - Shared Compilation Cache
 ==================================
 
-Sccache is a [ccache](https://ccache.samba.org/)-like tool. It is used as a compiler wrapper and avoids compilation when possible, storing a cache in a remote storage using the S3 API.
+Sccache is a [ccache](https://ccache.samba.org/)-like tool. It is used as a compiler wrapper and avoids compilation when possible, storing a cache in a remote storage using the S3 API, the GCS API, or Redis.
```

Comment:
I know we don't spell out S3 here, but could you expand GCS? I didn't know what it was at first glance. It'd be fine to explicitly say "Amazon S3" as well. Making those names into links to their respective home pages would be a good addition at this point.